doc/dev/index.rst

```diff
@@ -1132,11 +1132,12 @@ Reducing the number of tests
 ----------------------------
 
 The ``rados`` suite generates thousands of tests out of a few hundred
-files. For instance, all tests in the `rados/thrash suite
```

Comment:
This happens because teuthology constructs test matrices from subdirectories wherever it encounters a file named ``%``.